include/pm/pm.h

```diff
@@ -209,8 +210,8 @@ void pm_power_state_exit_post_ops(struct pm_state_info info);
 #define pm_constraint_release(pm_state)
 #define pm_constraint_get(pm_state) (true)
 
-#define pm_power_state_set(info)
-#define pm_power_state_exit_post_ops(info)
+#define pm_power_state_set(state, substate_id)
+#define pm_power_state_exit_post_ops(state, substate_id)
 #define pm_power_state_next_get(cpu) \
 	((struct pm_state_info){PM_STATE_ACTIVE, 0, 0})
```

Comment:
function returns a reference now, this needs to be updated.

Reply:
Oh, nicely spotted.